server/runtime/src/test/java/org/infinispan/server/test/ContainerInfinispanServerDriver.java

```diff
@@ -50,6 +52,7 @@
    CountdownLatchLoggingConsumer latch;
    ImageFromDockerfile image;
    private File rootDir;
+   private final boolean preferContainerExposedPorts = Boolean.valueOf(System.getProperty("org.infinispan.test.server.container.preferContainerExposedPorts", "false"));
```

Comment:
You can replace this with Boolean.getBoolean("org.infinispan.test.server.container.preferContainerExposedPorts")